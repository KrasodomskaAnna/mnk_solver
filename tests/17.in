SOLVE_GAME_STATE 3 4 3 1
1 2 0 0
0 0 0 0
0 0 0 0