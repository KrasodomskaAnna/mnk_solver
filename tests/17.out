FIRST_PLAYER_WINS
